src/multicloud-connector/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+1.0.0b1
```

Comment:
I'm guessing 1.0.0.b1 is a beta version.
For GA, we should get rid of the beta version.